ModAssistant/Pages/Mods.xaml.cs

```diff
@@ -693,8 +693,15 @@ public void UninstallMod(Mod mod)
                     break;
                 }
             }
-            if (mod.name.ToLower() == "bsipa")
-                UninstallBSIPA(links);
+            if (mod.name.ToLower() == "bsipa") { 
+                if (Directory.Exists(Path.Combine(App.BeatSaberInstallDirectory, "IPA")) & File.Exists(Path.Combine(App.BeatSaberInstallDirectory, "IPA.exe"))){
```

Comment:
```suggestion
                if (Directory.Exists(Path.Combine(App.BeatSaberInstallDirectory, "IPA")) && File.Exists(Path.Combine(App.BeatSaberInstallDirectory, "IPA.exe"))){
```

Reply:
This should be a conditional AND, not a logical AND

Reply:
Shouldn't it check for both?

Reply:
Yes, while a logical AND (single &) *will* work, it's not best practice. A conditional AND (double &&) is recommended to minimise introducing unintended behaviour.